### PR TITLE
fix: get offset from user

### PR DESCRIPTION
**Pull request by Olensavchenko**

### CODE REVIEW

File: packages/translation/pipes/xm-date-time.pipe.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class XmDateTimePipe implements PipeTransform {
     }
 
     private getOffset(): string {
-        return this.account?.timeZoneOffset || '';
+        return this.principal.getTimezoneOffset() || '';
```
Comment (Bozonych) [inline packages/translation/pipes/xm-date-time.pipe.ts:44]: Ir`s overhead. getTimezoneOffset() - now returns empty string.

**Review by Bozonych — APPROVED** [review]

File: packages/core/user/src/principal.service.ts [hunk 2]
```diff
@@ -266,6 +266,11 @@ export class Principal implements OnDestroy, OnInitialize {
         return moment().format('Z');
     }
 
+    public getTimezoneOffset(): string {
+        // For now setting offset from browser
+        return this.userIdentity.timeZoneOffset || '';
```
Comment (Bozonych) [inline packages/core/user/src/principal.service.ts:271]: Maybe make sense check this.userIdentity**?**.timeZoneOffset ??

File: packages/translation/pipes/xm-date-time.pipe.ts (hunk 1 above)
Comment (Olensavchenko) [reply packages/translation/pipes/xm-date-time.pipe.ts]: overhead deleted

File: packages/core/user/src/principal.service.ts (hunk 2 above)
Comment (Olensavchenko) [reply packages/core/user/src/principal.service.ts]: Yes. It makes sense.

**Review by Kaskyi — APPROVED** [review]

**Review by LevBD — APPROVED** [review]

**Pull request merged**